examples/secure_cloud_function_with_sql/providers.tf

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+provider "google" {
+  impersonate_service_account = var.terraform_service_account
+  request_timeout             = "60s"
+}
+
+provider "google-beta" {
+  impersonate_service_account = var.terraform_service_account
+  request_timeout             = "60s"
+}
```

Comment:
nit: do we need to explicitly call out providers in this example?
see https://cloud.google.com/docs/terraform/best-practices-for-terraform#providers-backends